### PR TITLE
DEP version 5.0.0 package updates

### PR DESCRIPTION
**Pull request by elsander**
This is a continuation of the work in #64, with the branch renamed to reflect a major version increment rather than minor version.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,60 @@ Version number changes (major.minor.micro) in this package denote the following:
...
+- cython 0.27.3 -> 0.29.6
+- dask 0.17.2 -> 1.1.4
+- ipython 6.1.0 -> 7.3.0
+- ipywidgets 7.1.0 -> 7.4.2
+- jinja2 2.9.6 -> 2.10
+- jsonschema 2.5.1 -> 3.0.1
+- libtiff 4.0.6 -> 4.0.10
+- libxml2 2.9.2 -> 2.9.8
+- matplotlib 2.2.2 -> 3.0.3
+- nomkl 1.0 -> 3.0
```
Comment (stephen-hoover) [inline CHANGELOG.md]: Was this a mistake? I don't see any change in `nomkl` (and it's not the sort of thing that should change).

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,60 @@ Version number changes (major.minor.micro) in this package denote the following:
...
+- pandas 0.22.0 -> 0.24.1
+- patsy 0.4.1 -> 0.5.1
+- psycopg2 2.6.2 -> 2.7.7
+- pyarrow 0.8.0 -> 0.12.1
+- pytest 3.1.3 -> 4.3.0
+- pyyaml 3.12 -> 3.13
+- requests 2.18.4 -> 2.21.0
+- s3fs 0.1.2 -> 0.2.0
+- seaborn 0.8 -> 0.9.0
+- scipy 1.0.1 -> 1.2.1
```
Comment (stephen-hoover) [inline CHANGELOG.md]: The `environment.yml` is updating `scipy` to 1.2.0.

**Review by stephen-hoover — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (elsander) [reply CHANGELOG.md]: Good catch, I'll fix it.

File: CHANGELOG.md (hunk 1 above)
Comment (elsander) [reply CHANGELOG.md]: The latest version on conda-forge is 3.0, although it looks like I didn't change the version number in the environment file. I don't have to update it, though.

File: CHANGELOG.md (hunk 2 above)
Comment (elsander) [reply CHANGELOG.md]: (I originally tried 1.2.1, but ran into version incompatibilities with libgfortran, so I'm sticking with 1.2.0 for now)

**Review by stephen-hoover — APPROVED** [review]
LGTM!

**Pull request merged**